api/v1alpha1/limitador_types.go

```diff
@@ -241,6 +194,44 @@ type RedisCached struct {
 	Options *RedisCachedOptions `json:"options,omitempty"`
 }
 
+// PersistentVolumeClaimResources defines the resources configuration
+// of the backup data destination PersistentVolumeClaim
+type PersistentVolumeClaimResources struct {
+	// Storage Resource requests to be used on the PersistentVolumeClaim.
+	// To learn more about resource requests see:
+	// https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+	Requests resource.Quantity `json:"requests"` // Should this be a string or a resoure.Quantity? it seems it is serialized as a string
+}
+
+type PVCGenericSpec struct {
+	// +optional
+	StorageClassName *string `json:"storageClassName,omitempty"`
+	// Resources represents the minimum resources the volume should have.
+	// Ignored when VolumeName field is set
+	// +optional
+	Resources *PersistentVolumeClaimResources `json:"resources,omitempty"`
+	// VolumeName is the binding reference to the PersistentVolume backing this claim.
+	// +optional
+	VolumeName *string `json:"volumeName,omitempty"`
+}
+
+// DiskOptimizeType defines the valid options for "optimize" option of the disk persistence type
+// +kubebuilder:validation:Enum=throughput;disk
+type DiskOptimizeType string
+
+const (
+	DiskOptimizeTypeThroughput DiskOptimizeType = "throughput"
+	DiskOptimizeTypeDisk       DiskOptimizeType = "disk"
+)
+
+type DiskSpec struct {
+	// +optional
+	PVC *PVCGenericSpec `json:"persistentVolumeClaim,omitempty"`
+
+	// +optional
+	Optimize *DiskOptimizeType `json:"optimize,omitempty"`
```

Comment:
maybe we can add 
`// +kubebuilder:default=throughput`

Reply:
I would rather rely on Limitador's own defaults. If optimize is not set in the CR, it would not go in the command line and defaults would be applied by Limitador, not Limitador Operator.